docs/usage/control-plane-endpoints-and-ports.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the reversed VPN tunnel, there are no endpoints with open ports in the shoot cluster required by gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are shared endpoints for all shoots of a seed cluster.
+The main entry-point into the seed cluster is the loadbalancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
+
+The Istio ingress-gateway exposes the following TCP ports:
+
+* **443** for requests to the shoot API-server. The request is dispatched according to the set TLS SNI extension.
+* **8443** for requests to the shoot API-server via `api-server-proxy`. 
+* **8132** to establish the reversed VPN connection.
+
+## Kube API-server via SNI 
+
+![API-server via SNI](./images/api-server-sni.png)
```

Comment:
We could add the port translation if we want. (The load balancer port is not equal to the istio pod port.)

Reply:
I added the it to the picture. However, I'm not sure if it's not beyond the intended scope and rather confusing.